components/admin_console/admin_definition.jsx

```diff
@@ -544,6 +561,7 @@ const AdminDefinition = {
                     },
                 ],
             },
+            indicator: getRestrictedIndicator(true),
```

Comment:
We should be checking on the actual license before setting this to always `true` here. For example, the key icon should show also in self-managed for features that are currently not available in the their current license so it comes down to the same check we do to show the all the Feature Discovery screens (i.e. condition in line 547 above) -> If Feature Discovery page is visible, the restricted indicator should be visible (inverse of `isHidden` for Feature Discovery page)

Reply:
In the case of a "feature discovery" item, this needs to be `true` as it's not available, so no need for checks.
In the case of an available item, now perform the checks for a cloud starter license.